gcn.notices.fermi.gbm.Specific.avsc

```diff
@@ -0,0 +1,82 @@
+  {
+  "type": "record",
+  "namespace": "gcn.notices.fermi.gbm",
+  "name": "Specific",
```

Comment:
It's the GBM specific parameters that are not part of the core schema.  I thought it was weird to call it gbm.GBMSpecific.  It could be gbm.InstrumentSpecific?